src/components/Card/ProductCard.js

```diff
@@ -1,4 +1,7 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { useCart } from "../../context/cart-context";
+import "../../pages/ProductListing/Products.css";
```

Comment:
As this component is already getting imported in `productListing.js`, so it's not necessary to import the `CSS` file here too